GOVERNANCE.md

```diff
@@ -1,36 +1,87 @@
 # Governance
 
-`haskell-cryptography` (herein referred to as the "Organisation") is a GitHub organisation that aims to provide a support structure to open-source maintainers producing cryptography libraries.
+`haskell-cryptography` (herein referred to as the "Organisation") is a GitHub organisation that aims to provide a support structure to open-source maintainers producing cryptography projects.
 
-This document elaborates on administrative and operational aspects of the Organisation.
+The Organisation has the following goals:
+
+* Prevent situations where too few people are supporting too critical projects for the ecosystem.
+* Prevent having a very closed group of people having too much control over every aspect of a project.
```

Comment:
```suggestion
* Prevent having a closed group of people having too much control over every aspect of a project.
```

I'm not sure this is particularly clear; we're encouraging people to cake control of projects but not too much control?